starter_code/elevator.js

```diff
@@ -1,19 +1,64 @@
+const Person = require('./person.js');
 class Elevator {
-  constructor(){
-    this.floor      = 0;
-    this.MAXFLOOR   = 10;
-    this.requests   = [];
+  constructor() {
+    this.floor = 0;
+    this.MAXFLOOR = 10;
+    this.requests = [];
+    this.waitingList = [];
+    this.passengers = [];
+    this.direction = "up"
   }
 
-  start() { }
-  stop() { }
-  update() { }
-  _passengersEnter() { }
-  _passengersLeave() { }
-  floorUp() { }
-  floorDown() { }
-  call() { }
-  log() { }
+  start() {
+    this.intervalId = setInterval(() => { this.update(); }, 1000)
```

Comment:
puedes prescindir de las llaves this.intervalId = setInterval(() => this.update(), 1000)